apps/web/src/pages/quick-start/components/TestNotificationTrigger.tsx

```diff
@@ -38,6 +44,7 @@ export function TestNotificationTrigger() {
       successMessage('Template triggered successfully');
       showTemplateCreation();
     } catch (e: any) {
+      segment.track(OnBoardingAnalyticsEnum.EXCEPTION, e.message || e);
```

Comment:
No need to track this one, you can use Sentry.captureException here :)

Reply:
Good catch!